pkg/controller/pkg/revision/reconciler.go

```diff
@@ -217,6 +224,10 @@ func SetupProviderRevision(mgr ctrl.Manager, l logging.Logger, cache xpkg.Cache,
 	return ctrl.NewControllerManagedBy(mgr).
 		Named(name).
 		For(&v1alpha1.ProviderRevision{}).
+		Watches(&source.Kind{Type: &extv1.CustomResourceDefinition{}}, &handler.EnqueueRequestForOwner{OwnerType: &v1alpha1.ProviderRevision{}}).
+		Watches(&source.Kind{Type: &extv1beta1.CustomResourceDefinition{}}, &handler.EnqueueRequestForOwner{OwnerType: &v1alpha1.ProviderRevision{}}).
```

Comment:
Do we need to watch both of these versions? I'm fairly sure that when one changes we'll see an update for the other.

Reply:
You are correct :slightly_smiling_face: 

> // OwnerType is the type of the Owner object to look for in OwnerReferences.  Only Group and Kind are compared.

Reply:
I see! So should we remove `extv1beta1` here?